spec/inputs/graphite_spec.rb

```diff
@@ -1 +1,54 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/inputs/graphite"
+require "logstash/timestamp"
+require "time"
+
+describe LogStash::Inputs::Graphite do
+  before do
+    srand(RSpec.configuration.seed)
+  end
+
+  let(:host) { "127.0.0.1" }
+  let(:port) { rand(5000) + 1025 }
+  let(:queue) { [] }
+
+  let(:client) { TCPSocket.new(host, port) }
+
+  subject { LogStash::Inputs::Graphite.new("host" => host, "port" => port) }
```

Comment:
The biggest reason is to not run code that is not related to this plugin. Using that will make this test depend on the pipeline